detection-rules/attachment_legal_notice_phishing_with_file_analysis.yml

```diff
@@ -0,0 +1,79 @@
+name: "Attachment: Legal Notice Phishing with Suspect PDF"
+description: |
+  Detects legal notice phishing attempts with PDF attachments containing suspicious URLs.
+  Uses file.explode to analyze PDF attachments for embedded URLs and runs link analysis
+  to identify credential phishing indicators in legal/threatening themed messages.
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and (
+    // subject contains legal/threatening language
+    regex.icontains(subject.subject,
+                    '\b(summon|subpoena|lawsuit|litigation|legal\s+action|court\s+order|warrant|violation|fine|penalty|demand\s+notice|legal\s+notice|court\s+notice|judgment|tribunal|hearing|proceedings|prosecution|contempt|breach|default|liability|damages|settlement|injunction)\b'
+    )
+    or strings.icontains(subject.subject, 'legal')
+    or strings.icontains(subject.subject, 'court')
+    or strings.icontains(subject.subject, 'summon')
+    or strings.icontains(subject.subject, 'notice')
+  )
+  and (
+    // external sender (not from trusted legal domains)
+    sender.email.domain.root_domain not in $org_domains
+    and sender.email.domain.root_domain not in $high_trust_sender_root_domains
+  )
```

Comment:
I'm a little confused, you have this snippet without DMARC checks, and then a non-standard high trust snippet at the end of your rule. Is there a reason for that?